shell.nix

```diff
@@ -7,7 +7,7 @@
 #   $ nix-shell --arg fast true
 #
 { fast ? false
-, compiler ? "ghc948"
+, compiler ? "ghc9101"
```

Comment:
How come we are updating this?

Reply:
The nix-shell environment doesn't work for 9.4.8.  Some problem with the `unix` package, IIRC.

Reply:
Gotcha, I'm looking into that now. Hence, #275.